BLSQ/iaso
# IA-2572: Organisation unit groups management not available in list of permissions

Explain what problem this PR is resolving
- The organisation unit groups management not available in list of permissions
- This PR is about to add it in the default module so that it can be accessible
Related JIRA tickets : [IA-2572](https://bluesquare.atlassian.net/browse/IA-2572)

## Self proofreading checklist


## Changes
- Added the iaso_org_unit_groups in the default module
- Added the tooltip to explain the use of this permission

## How to test
- Edit or create a user and check if the permission is in the list of permissions

## Print screen / video
![Screenshot from 2023-12-18 18-06-53](https://github.com/BLSQ/iaso/assets/19631540/ee1ae2ac-1f23-450c-ad50-c73b87273dd6)


[IA-2572]: https://bluesquare.atlassian.net/browse/IA-2572?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
